src/main/java/Ship/ShipManager.java

```diff
@@ -40,18 +41,36 @@ public void Draw(Graphics g) {
 		g.fillPolygon(myShip.getShape());
 	}
 	
+	
+	//needed to get ship health for the top bar
+	public int getShipHealth() {
```

Comment:
I would suggest renaming this to getHealth because it's already in the ship object. Also, this night be good being out in the spaceManager abstract class in the future since it's generic